docs/en/about/index.md

```diff
@@ -8,6 +8,11 @@ Thank you to [Andrew Byrd](https://www.linkedin.com/in/byrdandrew) for purchasin
 
 To propose edits to the content of this website, contact MobilityData at [documentation@mobilitydata.org](mailto:documentation@mobilitydata.org) or visit the [GTFS.org GitHub Repository](https://github.com/mobilitydata/gtfs.org) to submit your edits as pull requests.
 
+Regarding translations, GTFS.org uses an automated tool to ensure translations are kept up to date with repository changes. Since our priority is to keep translations up-to-date, any one-time translation contribution will be overwritten in the future. 
```

Comment:
I would say instead of "any one-time translation contribution will be overwritten in the future." : any translation corrections done manually on any language will be overwritten at some point in the future.

Reply:
Thanks Fred, would this text with slight adjustments work for you?

> "any translation corrected manually in any language will eventually be overwritten."

Reply:
Yeah, sounds good. Simpler as well.

Reply:
Thanks! the commit has been pushed, do you mind approving? :) thanks!